backward.hpp

```diff
@@ -3945,6 +4137,7 @@ class SignalHandling {
 #else
 #warning ":/ sorry, ain't know no nothing none not of your architecture!"
 #endif
+    st.set_context(_ctx, info->si_addr);
     if (error_addr) {
       st.load_from(error_addr, 32);
```

Comment:
I think it makes more sense to passe the context to `load_from`.

Reply:
I agree. I wanted to not be disruptive and change the public API, in case somebody upgraded backward.hpp and saw their code break. Do I add the new parameters to load_from then? Make them optional just in case?